consensus/reactor.go

```diff
@@ -1265,12 +1265,14 @@ func (ps *PeerState) SetHasVote(vote *types.Vote) {
 }
 
 func (ps *PeerState) setHasVote(height int64, round int32, voteType tmproto.SignedMsgType, index int32) {
-	logger := ps.logger.With(
-		"peerH/R",
-		fmt.Sprintf("%d/%d", ps.PRS.Height, ps.PRS.Round),
-		"H/R",
-		fmt.Sprintf("%d/%d", height, round))
-	logger.Debug("setHasVote", "type", voteType, "index", index)
+	if ps.logger.IsDebugging() {
+		logger := ps.logger.With(
```

Comment:
I think the logger.With should probably just be a bunch of fields (it's equivalent, I think). 

I think the fmt.sprintfs could be decomposed into fields, which would obviate the problem. 

My gut feeling is that this is probably the log statement in the hottest code path, so probably just deleting it would resolve the performance issue.

Reply:
Yeah.. instead of instantiating `.With` I'm just `log.LazySprintf`'ing